src/SignIn/SignIn.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import authService from '../firebase/authService';
+
+class SignIn extends Component {
+    constructor() {
+        super();
```

Comment:
Does this call a default constructor from the base Component class? Should we always be calling this?

Reply:
Honestly, not sure if this is necessary